flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -380,23 +249,23 @@ const BAG_WEIGHT = 20;
      * Calculate %MAC ZWFCG of all stations
      */
 function getZfwcg() {
-    const currentPaxWeight = PAX_WEIGHT + BAG_WEIGHT;
+    const currentPaxWeight = PAX_WEIGHT;
 
-    const leMacZ = -5.39; // Value from Debug Weight
-    const macSize = 13.45; // Value from Debug Aircraft Sim Tunning
+    const leMacZ = -5.386; // Accurate to 3 decimals, replaces debug weight values
+    const macSize = 13.454; // Accurate to 3 decimals, replaces debug weight values
 
-    const emptyWeight = 101990 * 0.453592; // Value from flight_model.cfg to kgs
+    const emptyWeight = 93697 * 0.453592; // Value from flight_model.cfg to kgs
```

Comment:
Can the empty weight be read from the empty weight Simvar rather than be hardcoded here? I'm concerned that someone in the future may want to change the empty weight, does so in the flight_model.cfg file, but doesn't realize it also needs to be changed here. Also, it would be good just for assurance that the same empty weight is being used in both places.

Reply:
Implemented feedback for both the places